app/assets/stylesheets/custom.css.scss

```diff
@@ -164,3 +164,32 @@ input, textarea, select, .uneditable-input {
   margin-left: 20px;
   font-weight: normal;
 }
+
+/*flash messages*/
+.alert-error {
+    background-color: #f2dede;
+    border-color: #eed3d7;
+    color: #b94a48;
+    text-align: left;
+ }
+
+.alert-alert {

```

Comment:
`.alert-alert` and `.alert-error` is similar
you can define it once, see => http://stackoverflow.com/a/2099263/2819006